betterjails/src/main/java/io/github/emilyydev/betterjails/BetterJailsPlugin.java

```diff
@@ -200,16 +210,15 @@ public void onEnable() {
 
     PlayerListeners.create(this).register();
 
-    final CommandHandler commandHandler = new CommandHandler(this);
-    final CommandTabCompleter tabCompleter = new CommandTabCompleter(this);
-    for (final String commandName : getDescription().getCommands().keySet()) {
-      final PluginCommand command = getCommand(commandName);
-      if (command != null) {
-        command.setExecutor(commandHandler);
-        command.setTabCompleter(tabCompleter);
-      }
+    final LegacyPaperCommandManager<CommandSender> commandManager =
+        LegacyPaperCommandManager.createNative(this, ExecutionCoordinator.simpleCoordinator());
```

Comment:
This fails on MockBukkit btw
`java.lang.NoClassDefFoundError: com/mojang/brigadier/exceptions/CommandSyntaxException`